pdns/dnsname.hh

```diff
@@ -149,7 +149,7 @@ public:
   DNSName& operator+=(const DNSName& rhs)
   {
     if(d_storage.size() + rhs.d_storage.size() > 256) // one extra byte for the second root label
-      throw std::range_error("name too long");
+      throwSafeRangeError("name too long", rhs.d_storage.data(), rhs.d_storage.size());
```

Comment:
This one could be a bit confusing since `rhs.d_storage` might not be too long by itself, but getting that information is better than none at all.

Reply:
Maybe "resulting name too long"?

Reply:
Sounds good!